kairon/events/executors/callback_executor.py

```
@@ -0,0 +1,28 @@
+            "task_type": kwargs.get("task_type", "Callback"),
+        }
+
+        try:
+            response = Utility.execute_http_request(
+                "POST",
+                  callback_url,payload
+            )
+            return response
+
+        except requests.RequestException as req_err:
+            raise AppException(f"Callback request failed: {str(req_err)}")
```

Ruff (0.8.2)

Within an `except` clause, raise exceptions with `raise ... from err` or `raise ... from None` to distinguish them from errors in exception handling

(B904)